UM/Settings/Models/InstanceContainersModel.py

```diff
@@ -250,4 +252,4 @@ def _updateMetaData(self, container):
     def _onContainerLoadComplete(self, container_id):
         if container_id in self._instance_containers_metadata:
             del self._instance_containers_metadata[container_id]
-            self._instance_containers[container_id] = ContainerRegistry.getInstance().findContainers(id = container_id)
\ No newline at end of file
+            self._instance_containers[container_id] = ContainerRegistry.getInstance().findContainers(id = container_id)[0]
```

Comment:
I see this [0] quite often. It may be an idea (for the future) to make a findContainer, which is special casing findContainers when you expect exactly 1 result

Reply:
We have it all over our code since these find functions exist. To me it's just a warning that you might get an empty list so you must be sure that the filter returns something. For instance in this case, it is first checked that `container_id` is in the model and it is guaranteed that anything in the model already exists.